.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,27 @@
-- [ ] Is it mergeable?
-- [ ] `make test` Did it pass the tests?
-- [ ] `make clean diff-cover` If it introduces new functionality in
-  `scripts/` is it tested?
-- [ ] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
-  formatted?
```

Comment:
Any `pep8` violations will cause a CI build failure. Anything beyond this hasn't been well-enforced up to this point.